python/call_synthesize.py

```diff
@@ -100,12 +111,17 @@ def internal_synthesize_streaming(stub, request, timeout, metadata, audio_saver)
                     raise RuntimeError("Sample rate does not match previously received.")
             else:
                 audio_saver.setFrameRate(response.audio.sample_rate_hertz)
+            if audio_player is not None:
+                audio_player.append(response.audio.content)
```

Comment:
shoudln't there be also 

audio_saver.setFrameRate(response.audio.sample_rate_hertz)

somewhere ?